packages/sakuli-forwarder-check_mk/src/checkmk-forwarder-properties.class.ts

```diff
@@ -60,4 +60,12 @@ export class CheckMkForwarderProperties {
     @BooleanProperty("sakuli.forwarder.check_mk.output.details")
     @IsBoolean()
     outputDetails: boolean = true;
+
+    @BooleanProperty("sakuli.forwarder.check_mk.url.enabled")
+    @IsBoolean()
+    urlEnabled: boolean = false;
+
+    @StringProperty("sakuli.forwarder.check_mk.url.name")
```

Comment:
```suggestion
    @StringProperty("sakuli.forwarder.check_mk.url")
```
I'd just remove the name as  URL stands for "Uniform Resource Locator". Maybe one could discuss to name it `sakuli.forwarder.check_mk.url.value` but I think shorter is better.